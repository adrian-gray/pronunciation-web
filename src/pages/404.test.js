import React from "react";
import { shallow } from "enzyme";
import { render, cleanup } from "@testing-library/react";

import FourOhFour from "./404";

afterEach(cleanup);

test("FourOhFour", async () => {
  const props = {};

  const container = shallow(<FourOhFour {...props} />);

  expect(container.firstChild).toMatchSnapshot();
});
