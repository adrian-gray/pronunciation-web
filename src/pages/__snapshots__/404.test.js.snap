// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FourOhFour 1`] = `
<div>
  <div
    class="head-space"
  >
    <h3>
      Oops, that address is not a thing I understand.
    </h3>
    <p>
      <a
        href="/"
      >
        Head to the homepage and try again.
      </a>
    </p>
  </div>
</div>
`;
